<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>upload file</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>571</width>
           <height>114</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="PathChoose">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>chose file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PathShow"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>local ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uploadLocalIP"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>server ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="uploadServerIP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>trans mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="uploadMode">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>netascii</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>octet</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="uploadButton">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>upload</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>download file</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>581</width>
           <height>111</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>server ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadServerIP"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>server filename</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadServerFilename"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>local ip </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadLocalIP"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>local filename </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="downloadLocalFilename"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>trans mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="downloadMode">
              <item>
               <property name="text">
                <string>netascii</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>octet</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="downloadButton">
              <property name="text">
               <string>download</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="output">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
